spanner/api/SpannerTest/Tests.cs

```diff
@@ -96,19 +101,14 @@ public void Dispose()
             catch (RpcException ex) when (ex.Status.StatusCode == StatusCode.NotFound) { }
         }
 
-        public string ProjectId { get; private set; } =
-            Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+        public string ProjectId { get; private set; } = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
         // Allow environment variables to override the default instance and database names.
         public string InstanceId { get; private set; } =
             Environment.GetEnvironmentVariable("TEST_SPANNER_INSTANCE") ?? "my-instance";
-        private static readonly string s_randomDatabaseName = "my-db-"
-            + TestUtil.RandomName();
-        private static readonly string s_randomToBeCancelledBackupName = "my-backup-"
-            + TestUtil.RandomName();
-        private static readonly string s_randomRestoredDatabaseName = "my-db-"
-            + TestUtil.RandomName();
-        public string DatabaseId =
-            Environment.GetEnvironmentVariable("TEST_SPANNER_DATABASE") ?? s_randomDatabaseName;
+        private static readonly string s_randomDatabaseName = $"my-db-{DateTime.UtcNow.ToTimestamp().Seconds}";
+        private static readonly string s_randomToBeCancelledBackupName = $"my-backup-{DateTime.UtcNow.ToTimestamp().Seconds}";
+        private static readonly string s_randomRestoredDatabaseName = $"my-db-{DateTime.UtcNow.AddMinutes(1).ToTimestamp().Seconds}";
+        public string DatabaseId = Environment.GetEnvironmentVariable("TEST_SPANNER_DATABASE") ?? s_randomDatabaseName;
         public string BackupDatabaseId = "my-test-database";
         public string BackupId = "my-test-database-backup";
         public string ToBeCancelledBackupId = s_randomToBeCancelledBackupName;

```

Comment:
Why don't these need to be random? Just asking, I'm fine if there's no need for it.